curriculum/challenges/english/24-b1-english-for-developers/learn-present-perfect-while-talking-about-accessibility/672e1c3031816ceb9dccc66a.md

```diff
@@ -0,0 +1,102 @@
+---
+id: 672e1c3031816ceb9dccc66a
+title: Task 112
+challengeType: 19
+dashedName: task-112
+---
+
+<!-- (Audio) James: We're working on it. The instructional design team has been brainstorming ideas to make the sessions more interactive. We're also considering incorporating more real-life scenarios. -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+What might the team do next, since James mentions they are `also considering incorporating more real-life scenarios`?
+
+## --answers--
+
+Add activities that focus on memorizing definitions.
+
+### --feedback--
+
+Memorizing definitions doesn’t involve real-life application or hands-on practice, which James implies they are aiming for.
+
+---
+
+Use theoretical case studies with no practical application.
+
+### --feedback--
+
+Theoretical case studies alone don’t fully connect to real-life tasks, which James wants to include.
+
+---
+
+Have participants discuss unrelated topics to build general communication skills.
+
+### --feedback--
+
+Unrelated discussions don’t meet the goal of incorporating relevant, real-life situations.
+
+---
+
+Create exercises that use realistic situations participants might encounter on the job.
+
+## --video-solution--
+
+4
+
+# --explanation--
+
+Since James mentions `incorporating more real-life scenarios`, they’re likely planning to add exercises that reflect situations participants could actually encounter in their roles. This makes the training more engaging and relevant, unlike focusing on definitions, unrelated discussions, or purely theoretical case studies.
+
+# --scene--
+
+```json
+{
+  "setup": {
+    "background": "interview-room1.png",
+    "characters": [
+      {
+        "character": "James",
+        "position": {
+          "x": 50,
+          "y": 15,
+          "z": 1.2
+        },
+        "opacity": 0
+      }
+    ],
+    "audio": {
+      "filename": "B1_4-4.mp3",
+      "startTime": 1,
+      "startTimestamp": 40.94,
+      "finishTimestamp": 50.32
+    }
+  },
+  "commands": [
+    {
+      "character": "James",
+      "opacity": 1,
+      "startTime": 0
+    },
+    {
+      "character": "James",
+      "startTime": 1,
+      "finishTime": 10.38,
+      "dialogue": {
+        "text": "We're working on it. The instructional design team has been brainstorming ideas to make the sessions more interactive. We've also considered incorporating more real-life scenarios.",
+        "align": "center"
+      }
+    },
+    {
+      "character": "James",
+      "opacity": 0,
+      "startTime": 10.88
+    }
+  ]
+}
+```
```

Comment:
```suggestion
# --scene--

```json
{
  "setup": {
    "background": "interview-room1.png",
    "characters": [
      {
        "character": "James",
        "position": {
          "x": 50,
          "y": 15,
          "z": 1.2
        },
        "opacity": 0
      }
    ],
    "audio": {
      "filename": "B1_4-4.mp3",
      "startTime": 1,
      "startTimestamp": 40.64,
      "finishTimestamp": 50.42
    }
  },
  "commands": [
    {
      "character": "James",
      "opacity": 1,
      "startTime": 0
    },
    {
      "character": "James",
      "startTime": 1,
      "finishTime": 2.08,
      "dialogue": {
        "text": "We're working on it.",
        "align": "center"
      }
    },
    {
      "character": "James",
      "startTime": 2.09,
      "finishTime": 7.08,
      "dialogue": {
        "text": "The instructional design team has been brainstorming ideas to make the sessions more interactive.",
        "align": "center"
      }
    },
    {
      "character": "James",
      "startTime": 7.09,
      "finishTime": 10.78,
      "dialogue": {
        "text": "We've also considered incorporating more real-life scenarios.",
        "align": "center"
      }
    },
    {
      "character": "James",
      "opacity": 0,
      "startTime": 11.28
    }
  ]
}
```
```